### DISCUSSION TITLE
Animated Splash Screen

### DISCUSSION THREAD

**Discussion by jrahma** [xamarin/Xamarin.Forms#3729] [General]
Hi,

Is the any plan to add an option which allows animated splash screen for iOS, Android and UWP?

Thanks
Jassim

**Comment by GalaxiaGuy** [comment]
The iOS launch screen is required to be static since it is handled by the system. You could make the first page of the app look the same as the launch screen (initially) and then do whatever you want, including animation. At that point it is unclear what extra functionality you are asking for.

There are a few ways splash screens on Android are implemented and, somewhat like iOS, they have to be implemented outside of Forms so they get displayed while Forms is loading.

I don't really know much about other platforms but I suspect they use on of those two methods.

**Comment by adrianknight89** [comment]
@jrahma As far as I'm aware, there is no such thing as an animated splash screen (other people can correct me on this). You can, however, put up a static image like @GalaxiaGuy suggested and then run an animation to trick people into thinking they are looking at an animated splash screen. Maybe, [LottieXamarin](https://github.com/martijn00/LottieXamarin) can be of good use for this purpose.

**Comment by candidodmv** [comment]
any updates about this very interesting feature?

**Comment by jrahma** [comment]
Keep in mind this should also provide an option to set the a splash screen for light mode and another splash screen for dark mode so all what user needs to do is just to select the image / video for each mode
